### PR TITLE
Bug 1757056, update KubeletConfig CRD to modify maxPods procedure

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1757056

Preview Build: http://file.rdu.redhat.com/~ahardin/10092019/modifying-maxpods/scalability_and_performance/recommended-host-practices.html#create-a-kubeletconfig-crd-to-edit-kubelet-parameters_

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@wabouhamad @chaitanyaenr Can you please review and let me know if I missed anything? Thanks!

File: modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc [hunk 1]
```diff
@@ -44,49 +44,63 @@ Allocatable:
...
 ----
 
-By default, only one machine is allowed to be unavailable when applying the kubelet-related configuration to the available worker nodes. For a large cluster, it can take a long time for the configuration change to be reflected. At any time, you can adjust the number of machines that are updating to speed up the process.
-
-.Procedure
-. Run:
+.. Run:
 +
 ----
-$ oc edit machineconfigpool worker
```
Comment (chaitanyaenr) [inline modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc]: @ahardin-rh can we keep the procedure to change the number of maxUnavailable nodes at a given time when rebooting the nodes to apply the kubelet config changes please?

Comment (wabouhamad) [reply modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc]: @ahardin-rh or perhaps put link to the procedure to change the number of maxUnavailable nodes at a given time when rebooting the nodes here.

File: modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc [hunk 2]
```diff
@@ -44,49 +44,63 @@ Allocatable:
...
-$ oc apply -f max-worker-pods.yaml
+$ oc label machineconfigpool worker custom-kubelet=large-pods
 ----
 
-. Verify the change by running:
+.. Run:
 +
 ----
-$ oc get kubeletconfigs set-max-pods -o yaml
+$ oc create -f worker-kube-config-change-maxPods-crd.yaml
```
Comment (wabouhamad) [inline modules/create-a-kubeletconfig-crd-to-edit-kubelet-parameters.adoc]: We could shorten the name of the CR may be to change-maxPods-cr.yaml ?

**Comment by ahardin-rh** [comment]
@wabouhamad I made some updates PTAL. The latest changes are copied to this preview build:

http://file.rdu.redhat.com/~ahardin/10092019/modifying-maxpods/scalability_and_performance/recommended-host-practices.html#create-a-kubeletconfig-crd-to-edit-kubelet-parameters_

**Review by chaitanyaenr — APPROVED** [review]
/lgtm

**Comment by wabouhamad** [comment]
LGTM.  Thank you @ahardin-rh !

**Comment by ahardin-rh** [comment]
Thank you!

**Pull request merged**